Extension/package.json

```diff
@@ -823,7 +823,14 @@
                         "scope": "resource"
                     },
                     "C_Cpp.default.browse.path": {
-                        "type": "array",
+                        "oneOf": [
+                            {
+                                "type": "array"
+                            },
+                            {
+                                "type": null
+                            }
+                        ],
                         "items": {
```

Comment:
The `items` property needs to go in the block where you set the type to `array`.

Reply:
`uniqueItems` probably needs to go in there too.